src/main/java/org/opensearch/security/dlic/rest/api/PermissionsInfoAction.java

```diff
@@ -46,12 +46,14 @@
 import org.opensearch.security.user.User;
 import org.opensearch.threadpool.ThreadPool;
 
+import com.google.common.collect.ImmutableList;
+
 /**
  * Provides the evaluated REST API permissions for the currently logged in user
  */
 public class PermissionsInfoAction extends BaseRestHandler {
-	private static final List<Route> routes = Collections.singletonList(
-			new Route(Method.GET, "/_opendistro/_security/api/permissionsinfo")
+	private static final List<Route> routes  = Collections.singletonList(
+			new Route(Method.GET, "/permissionsinfo")
```

Comment:
Have this align with `DashboardsInfoAction`, `SecurityHealthAction` etc.

Simply declare additional paths in the list